workflow/executor/emissary/emissary.go

```diff
@@ -43,6 +43,10 @@ The wait container can create one file itself, used for terminating the sub-proc
 */
 type emissary struct{}
 
+func (e *emissary) GetExitCode(ctx context.Context, containerName string) (string, error) {
```

Comment:
revert

Reply:
@sarabala1979 you seem to have a few merge issues in this PR, can I ask you to review and fix them?